### PR TITLE
Add a CVE-2023-0461 mitigation guide

### PR DESCRIPTION
**Pull request by alanm-hpe**
Create a mitigation guide for CVE-2023-0461. When the CVE is resolved by an official SUSE fix, this will be removed.

### CODE REVIEW

File: docs/portal/developer-portal/operations/CVE-2023-0461.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+> There is a use-after-free vulnerability in the Linux Kernel which can be exploited to achieve local privilege escalation. To reach the vulnerability kernel configuration flag CONFIG_TLS or CONFIG_XFRM_ESPINTCP has to be configured, but the operation does not require any privilege. There is a use-after-free bug of icsk_ulp_data of a struct inet_connection_sock. When CONFIG_TLS is enabled, user can install a tls context (struct tls_context) on a connected tcp socket. The context is not cleared if this socket is disconnected and reused as a listener. If a new socket is created from the listener, the context is inherited and vulnerable. The setsockopt TCP_ULP operation does not require any privilege. We recommend upgrading past commit 2c02d41d71f90a5168391b6a5f2954112ba2307c
+
+[CVE-2023-0461](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-0461)
+
+Current status from SUSE:
+https://www.suse.com/security/cve/CVE-2023-0461.html
+
+## Mitigation
+While the underlying CVE is being addressed by SUSE, UANs can mitigate this issue by unbonding the CAN (if it is being used), and unloading the TLS kernel module after blocking the kernel from being loaded again.
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/operations/CVE-2023-0461.md:11]:
```suggestion
## Mitigation

While the underlying CVE is being addressed by SUSE, UANs can mitigate this issue by unbonding the CAN (if it is being used), and unloading the TLS kernel module after blocking the kernel from being loaded again.
```

File: docs/portal/developer-portal/operations/CVE-2023-0461.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+1. Replace `bond0` in `ifcfg-can0` with the `BONDING_SLAVE0` interface
+1. Reload interfaces
+1. Add a blacklist for the tls module
+1. Unload `bonding` and `tls` kernel modules provided `mlx5_core` is not present
+1. Fail if `mlx5_core` is detected to highlight that the mititagtion failed
+
+*Important*: This mitigation is intend for UANs that meet the following criteria:
+* The UANs are connected to HPE Aruba Networking switches (`lcap-individual` must be set for other switch types to allow for unbonded CAN connections)
+* The UANs use HPE Slingshot HSN NICs and not Mellanox ConnectX-5 HSN NICs.
+## Procedure
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/operations/CVE-2023-0461.md:24]:
```suggestion
* The UANs use HPE Slingshot HSN NICs and not Mellanox ConnectX-5 HSN NICs.

## Procedure
```

File: docs/portal/developer-portal/operations/CVE-2023-0461.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+## Description
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/operations/CVE-2023-0461.md:1]:
```suggestion

# Mitigation for CVE-2023-0461

## Description
```

**Review by darrenn-jackson — CHANGES_REQUESTED** [review]
Needs a few minor changes, mostly just formatting fixes.

**Review by darrenn-jackson — APPROVED** [review]

**Review by keopp-hpe — APPROVED** [review]

**Pull request merged**